src/middleware.ts

```diff
@@ -261,7 +261,7 @@ export async function middleware(request: NextRequest) {
         );
     } else {
         if (resolved.cacheMaxAge) {
-            const cacheControl = `public, max-age=60, s-maxage=${resolved.cacheMaxAge}, stale-while-revalidate=60, stale-if-error=0`;
+            const cacheControl = `public, max-age=0, s-maxage=${resolved.cacheMaxAge}, stale-while-revalidate=60, stale-if-error=0`;
```

Comment:
Do you want to remove the `stale-while-revalidate` as well?

Otherwise it might still result in 60s, in the edge, but also on the client